modules/ROOT/pages/metrics/configuring-dev-env-metrics.adoc

```diff
@@ -11,8 +11,4 @@ include::{partialsdir}/attributes.adoc[]
 
 include::{partialsdir}/generic-obtaining-the-mobile-sdk-config-file.adoc[leveloffset=1]
 
-Metrics will be sent automatically from the Mobile Application once any of the components from {SDK} are installed.
-
-== Adding AeroGear SDK modules for the {service-name} Service
-
-//***TODO***
+Metrics will be sent automatically from the Mobile Application once any of the components from {SDK} are installed.
```

Comment:
"mobile app" is used a number of times in other areas - does it makes sense to replace "Mobile Application" with "mobile app" for consistency?